src/css/components/contents.scss

```diff
@@ -0,0 +1,28 @@
+@import '../override_vars.scss';
+@import '../core.scss';
+
+.contents-primary {
```

Comment:
I'm not really sure what `contents` means in this context. What is this component trying to accomplish? What's it's purpose?